src/ui/src/builder/journal/BuilderJournalBlockOutput.vue

```diff
@@ -0,0 +1,449 @@
+<template>
+	<div class="output-block" :class="getOutcomeClass(output.outcome)">
+		<div
+			class="output-block__header"
+			role="button"
+			tabindex="0"
+			@click="$emit('go-to-block', blockId)"
+			@keydown.enter="$emit('go-to-block', blockId)"
+		>
```

Comment:
Maybe you can simply use a button. I know it sounds weird, but it's better in term of accessibility
```suggestion
		<button
			class="output-block__header"
			type="button"
			@click="$emit('go-to-block', blockId)"
		>
```
It should work with small CSS rules. The only constraint of this solution is, you can't have nested `button` inside